libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -0,0 +1,73 @@
+import { getConfig } from '../../utils/utils.js';
+
+async function getKey(product) {
+  const config = getConfig();
+  let keyMatch = [];
+  const resp = await fetch(`${config.contentRoot ?? ''}/branch-io-key.json`);
+  if (resp.ok) {
+    const json = await resp.json();
+    keyMatch = json.data.filter(
+      (p) => p.product === product,
+    );
+  }
+  return keyMatch[0]?.key;
+}
+
+/* eslint-disable */
+function branchInit(header, key) {
+  let initValue = false;
+  function initBranch() {
+    if (initValue) {
+      return;
+    }
+    initValue = true;
+    (function (b, r, a, n, c, h, _, s, d, k) {
+      if (!b[n] || !b[n]._q) {
+        for (; s < _.length;) c(h, _[s++]);
+        d = r.createElement(a);
+        d.async = 1;
+        d.src = 'https://cdn.branch.io/branch-latest.min.js';
+        k = r.getElementsByTagName(a)[0];
+        k.parentNode.insertBefore(d, k);
+        b[n] = h;
+      }
+    })(
+      window,
+      document,
+      'script',
+      'branch',
+      function (b, r) {
+        b[r] = function () {
+          b._q.push([r, arguments]);
+        };
+      },
+      { _q: [], _v: 1 },
+      'addListener applyCode autoAppIndex banner closeBanner closeJourney creditHistory credits data deepview deepviewCta first getCode init link logout redeem referrals removeListener sendSMS setBranchViewData setIdentity track validateCode trackCommerceEvent logEvent disableTracking'.split(
+        ' '
+      ),
+      0
+    );
+    const privacyConsent = !!window.adobePrivacy && window.adobePrivacy.hasUserProvidedConsent();
+    branch.init(key, { tracking_disabled: !privacyConsent });
+    branch.addListener('didShowJourney', () => {
+      header.style.position = 'relative';
+    });
+    branch.addListener('didCloseJourney', () => {
+      header.style.position = 'sticky';
+    });
```

Comment:
I'm a fan of turning this to a one liner:
```suggestion
    branch.addListener('didShowJourney', () => header.style.position = 'relative');
    branch.addListener('didCloseJourney', () => header.style.position = 'sticky');
```

Depending on our linter rules, this might not be accepted though (I think it is)

Reply:
done